src/Cell.tsx

```diff
@@ -27,10 +27,8 @@ function Cell<R>({
   onRowDoubleClick,
   enableCellRangeSelection
 }: CellProps<R>) {
-  const position: Position = { idx, rowIdx };
-
   function selectCell(openEditor?: boolean) {
-    eventBus.dispatch(EventTypes.SELECT_CELL, position, openEditor);
+    eventBus.dispatch(EventTypes.SELECT_CELL, { idx, rowIdx }, openEditor);
```

Comment:
I changed this so selecting the same cell again would trigger [focus](https://github.com/adazzle/react-data-grid/blob/af1676eec78ba7433885869289a6769466ce0c38/src/masks/InteractionMasks.tsx#L107).